src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts

```diff
@@ -129,45 +129,30 @@ export class DataMapperHostComponent extends FlowPage
   }
 
   createDocumentDefinition(dataShape: DataShape, isSource: boolean = false) {
-    if (!dataShape || !dataShape.type || !dataShape.kind) {
+    if (!dataShape || !dataShape.kind) {
       // skip
       return;
     }
     const type = dataShape.type;
     const kind = dataShape.kind;
-    // avoid passing duplicates off to the data mapper
-    if (isSource) {
-      if (this.sourceDocTypes.find(d => d === type)) {
-        return;
-      }
-      this.sourceDocTypes.push(type);
-    } else {
-      if (this.targetDocTypes.find(d => d === type)) {
-        return;
-      }
-      this.targetDocTypes.push(type);
-    }
-    log.infoc(
```

Comment:
I think we want this log at least?

Reply:
I kinda find the logs noisy enough as it is :-)

Reply:
right? if you say so :)